src/main/java/duke/logic/parsers/ParserStorageUtil.java

```diff
@@ -145,4 +147,44 @@ public static TrainStation createTrainFromStorage(String line) {
         }
         return new TrainStation(trainCode, address, latitude, longitude);
     }
+
+    /**
+     * Returns a list of todo's from a text file.
+     *
+     * @return The List of todo's
+     */
+    public static List<Todo> getTodoListFromStorage(String line) {
+        List<Todo> todoList = new ArrayList<>();
+        String[] todoParts = line.split("\\|");
+        for (String todoPart : todoParts) {
+            Todo todo = new Todo(todoPart);
+            todoList.add(todo);
+        }
+        return todoList;
+    }
+
+    /**
+     * Returns the Day object in a string format.
+     *
+     * @return The Information for one day in a string format
+     */
+    public static String toDayString(Day day) {
```

Comment:
Is it possible to override the toString method?

Reply:
Yes , overdid the to string method.